docs/build/build-guides-coretime-troubleshoot.md

```diff
@@ -0,0 +1,63 @@
+---
+id: build-guides-coretime-troubleshoot
+title: Troubleshooting
+sidebar_label: Troubleshooting
+description: Introduction to the Polkadot SDK
+keywords: [coretime, blockspace, parathread, parachain, cores, coretime, agile]
+slug: ../build-guides-coretime-troubleshoot
+---
+
+This page aims to cover and aggregate various resources that relate to troubleshooting common
+problems when using the Polkadot SDK or deploying on a core.
+
+## FAQ / Troubleshooting
+
+### Why do I have to sync Rococo locally? Can't I just use a remote, trusted node and connect to that?
+
+**A:** You can actually, via the `--relay-chain-rpc-urls` flag, which can be passed to your
+collator. Unfortunately, the caveat is you can't use this for collation at this time - meaning if
```

Comment:
"You can actually" sentence incomplete and needs further context. Also remove ambiguity of referring the node as a collator node, because the next sentence says the collator can't be used for collation which makes no sense. I thought you meant a Rococo node which can't be used for collation.

```suggestion
**A:** You can remotely connect to Rococo network via the --relay-chain-rpc-urls flag, which can be passed to your node. Unfortunately, the caveat is you can't use this node for collation at this time - meaning if
```